ci/build_cub.sh

```diff
@@ -16,7 +16,10 @@ version_compare() {
 
 ENABLE_CUB_BENCHMARKS="false"
 ENABLE_CUB_RDC="false"
-if [[ "$CUDA_COMPILER" == *nvcc* ]]; then
+
+if [[ -n "${DISABLE_CUB_BENCHMARKS}" ]]; then
+    echo "Benchmarks have been forcefully disabled."
+elif [[ "$CUDA_COMPILER" == *nvcc* ]]; then
     ENABLE_CUB_RDC="true"
```

Comment:
important: I believe this accidentally disabled RDC tests, which is not related to CUB benchmarks.